merlin/models/tf/transformers/transforms.py

```diff
@@ -37,10 +37,62 @@ class LastHiddenState(Layer):
         The output class returned by the HuggingFace transformer layer
     """
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.supports_masking = True
+
     def call(self, inputs: TFBaseModelOutputWithPoolingAndCrossAttentions):
         return inputs.last_hidden_state
 
 
+@Block.registry.register("inference_hidden_state")
+@tf.keras.utils.register_keras_serializable(package="merlin.models")
+class InferenceHiddenState(Layer):
```

Comment:
I don't see in the PR where `InferenceHiddenState` is going to be used yet...

Reply:
It is used in the unit test using its string representation the parameter `post="inference_hidden_state"`